crates/cxx-qt-gen/test_outputs/naming.cpp

```diff
@@ -34,19 +34,20 @@ MyObject::qtThread() const
                                                m_rustObjMutex);
 }
 
-qint32
+const qint32&
```

Comment:
Later we may want to differentiate between trivial and non-trivial types, because a qint32 doesn't really benefit from being returned by reference.

Reply:
Right, the fun of trying to work out which are trivial is the main issue, maybe we can have an include list to match them or something. But for now keeping the generation consistent was simplest for the next stages...